src/coreclr/tests/src/Common/test_dependencies/test_dependencies.csproj

```diff
@@ -8,7 +8,7 @@
     <RuntimeIdentifiers>win-arm;win-arm64;win-x64;win-x86;$(TargetRid)</RuntimeIdentifiers>
     <TargetArchitecture>$(BuildArch)</TargetArchitecture>
     <IncludeOOBLibraries>true</IncludeOOBLibraries>
-    <LibrariesConfiguration>Release</LibrariesConfiguration>
+    <LibrariesConfiguration>$(Configuration)</LibrariesConfiguration>
```

Comment:
@jkoritzinsky - have you got any context why this needs changing? I don't see anything outright wrong here but it seems orthogonal to the task of enabling mono-specific test exclusions as such. @naricc, if this is a semantic change that is for whatever reason required for Mono builds, it should be made mono-conditional and ideally equipped with an explanatory comment why this logic differs between Mono and CoreCLR.

Reply:
I don't know why we'd want to change this. The CoreCLR tests should always use the Release build of Libraries (there's no reason for them to use debug).

Reply:
@trylek 

I found that if its hard coded for Release here, the build-test will always check that some files exist in /artifacts/...release... directory, even if you have built everything in debug. With these two changes, I can build in debug and everything works.   

I can move them to a seperate PR if you like, since they are orthoganal. But I would like to fix it, unless there is some reason not to. I found it really confusing when trying to come up to speed on this, and it will probably trip some one else up in the future.

Reply:
Alternatively, we could just make build-tests.sh not accept the -Debug argument; that would also remove the confusion.

Reply:
We want a way to be able to build the tests in Debug but still always use the release libraries. I'd rather have this change in a separate PR so more of the CoreCLR teams can chime in on how they would like this to work.

cc: @jashook

Reply:
Ok; I will take it out of this PR. It sounds like maybe a more comprehensive change is needed later.